### PR TITLE
[SCAL-89469] - Added version dropdown

### PR DESCRIPTION
**Pull request by harsh-ts**
Ticket Link: https://thoughtspot.atlassian.net/browse/SCAL-89469

### CODE REVIEW

File: docs/src/components/Dropdown/index.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        overflow: hidden;
+    }
+    
+    .dropdown .dropbtn {
+        font-size: $font-size-normal;
+        border: none;
+        outline: none;
+        color: $white;
+        background-color: inherit;
+        font-family: inherit;
```
Comment (jbhanu-thoughtspot) [inline docs/src/components/Dropdown/index.scss:15]: Any reason for using inherit especially here? shouldn't the properties already be inherited?

File: docs/src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            label: 'Cloud',
+            link: '/visual-embed-sdk/release/en',
+        },
+        {
+            label: 'Dev',
+            link: '/visual-embed-sdk/dev/en',
+        },
+        {
+            label: '7.1',
```
Comment (jbhanu-thoughtspot) [inline docs/src/components/Dropdown/index.tsx:18]: If we add a new version I guess we'll need to update all these files with the new addition of the branch. Let's add a config file, which will contain these release versions and the branch mapping, and we just iterated over that array here.

Comment (jbhanu-thoughtspot) [reply docs/src/components/Dropdown/index.tsx:18]: You can also use that common config file to be used on the scripts

File: docs/src/components/Dropdown/index.scss (hunk 1 above)
Comment (harsh-ts) [reply docs/src/components/Dropdown/index.scss:15]: for `font-family: inherit;` we will remove it but for `background-color` we will require this property

File: docs/src/components/Dropdown/index.tsx (hunk 2 above)
Comment (harsh-ts) [reply docs/src/components/Dropdown/index.tsx:18]: Will make changes as suggested

**Review by jbhanu-thoughtspot — APPROVED** [review]
lgtm

**Pull request merged**